packages/kotti-ui/source/kotti-navbar/components/NavbarMenu.vue

```diff
@@ -6,12 +6,13 @@
 				class="kt-navbar-menu__section"
 				v-text="isNarrow ? '' : item.title"
 			/>
-			<a
+			<component
+				:is="link.component ? link.component : 'a'"
```

Comment:
according to the prop schema `a` is default

but i see it as a fallback , not a Vue prop `default` in the parent of this?

Reply:
You can parse the received props with zod, which will populate your defaults. What you’re describing is a limitation of `makeProps` since `vue` doesn’t support changing data in the validator.